src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,4 +1,12 @@
 package frc.robot;
 
 public class RobotMap {
+    public interface CAN {
+        int TRANSFER_STRAP_TALON_1 = -1;
+        int TRANSFER_STRAP_TALON_2 = -1;
+    }
+
+    public interface DIO {
```

Comment:
blank line at the start of the interface

Reply:
ditto

Reply:
add please